packages/popover/src/positionUtils.ts

```diff
@@ -40,7 +40,11 @@ export function calculatePosition({
   contentElPos = defaultElementPosition,
   windowHeight = window.innerHeight,
   windowWidth = window.innerWidth,
-}: CalculatePosition) {
+}: CalculatePosition): {
+  alignment: Align;
+  justification: string;
```

Comment:
I feel like we should have an enum for `Justification`. I know why this is currently string, but it might be worth creating one.

Reply:
Done